### ISSUE TITLE
Narrower sidebar

### ISSUE THREAD

**Issue by lars-hellstrom** [OpenMath/OpenMath.github.io#42]
I think the sidebar should be narrower. (This is probably related to #15.)

To be more precise, the current sidebar eats up too much of the screen width. That doesn't matter much for short pages, but for long pages the difference to the old openmath.org layout is considerable.

In particular, I've been looking at the Technical Overview. When viewed with 
[the old openmath.org layout](https://github.com/OpenMath/OpenMath.github.io/files/1284374/OpenMath.A.Technical.Overview.OpenMath.pdf), it reads quite well – the reader can see that this page contains interesting content. When viewed with [the new OpenMath.github.io layout](https://github.com/OpenMath/OpenMath.github.io/files/1284378/OpenMath.github.io.A.Technical.Overview.OpenMath.pdf), the reader is instead bogged down in some rather uninteresting initial paragraphs about "Why XML?". (My browser doesn't include the sidebar in those PDFs for some reason, but here is a screendump
![screendump](https://user-images.githubusercontent.com/30267049/30159845-5ba82c56-93ca-11e7-850a-29ebae853364.png) 
of what it looks like; the PDF pagination is fairly close to the screen pagination.

There are probably other factors – such as font and font size – contributing to the difference, but I can't shake the impression that the sidebar is eating more than a third of the window, and contributing very little beyond the first screenful of actual page contents.

The CDs aren't hurt as much by this, probably because the main content has a fixed width or something, but I still find myself scrolling right as the first thing I do when I open one of those pages, simply because that gets the (mostly empty) "sidebar column" off my screen.

**Comment by kohlhase** [comment]
I agree with this sentiment. I made the fonts a little smaller; maybe that helps.

**Comment by davidcarlisle** [comment]
the relative width of sidebar and main text are not controlled by the fonts but by (as far as I can tell)
`public/css/hyde.css` having

`  width: 18rem;`  for the sidebar 

and

` max-width: 38rem;` for the content.
